conf/zapi/cdot/9.8.0/volume.yaml

```diff
@@ -63,6 +63,7 @@ counters:
       - ^auto-snapshots-enabled             => auto_snapshots_enabled
       - ^snapshot-policy
       - snapshot-count
+    - ^encrypt                              => isEncrypted
```

Comment:
is this counter aligned with the others?

Reply:
yes, it's direct under volume-get-iter.